epub34/a11y/index.html

```diff
@@ -1711,8 +1697,8 @@ <h2>Distribution</h2>
 
 			<div class="note">
 				<p>A distributor may implement a digital rights management scheme that inherently impairs accessibility
-					through no fault of the EPUB creator. Following the guidance in this section does not restrict EPUB
-					creators from using such distributors. The intent is only that the EPUB creator not impair
+					through no fault of the publisher. Following the guidance in this section does not restrict
```

Comment:
This note breaks my brain. I think maybe we need to rethink it - are we basically saying "the stuff that isn't your fault isn't your fault"? Can we say that without the word publisher? I don't even know who "you" is here - apparently it is not distributors, but what happens when a distributor creates or modifies your epub? It really seems like we are saying what happens downstream (by anyone) is not your fault, unless it was, in which case don't do that.

Reply:
Ya, I think this can be boiled down to:

> The guidance in this section does not restrict EPUB publications from being distributed through inaccessible distribution channels. For example, where the distributor applies DRM that impairs access to the content.

Reply:
Yeah, that actually makes sense to me. 👍